packages/ERTP/src/issuerKit.js

```diff
@@ -24,32 +24,29 @@ import './types.js';
  *
  *  `displayInfo` gives information to the UI on how to display the amount.
  *
- * @param {string} allegedName
- * @param {K} [assetKind=AssetKind.NAT]
- * @param {AdditionalDisplayInfo} [displayInfo={}]
+ * @param {MapStore<string,any>} issuerBaggage
  * @param {ShutdownWithFailure=} optShutdownWithFailure If this issuer fails
  * in the middle of an atomic action (which btw should never happen), it
  * potentially leaves its ledger in a corrupted state. If this function was
  * provided, then the failed atomic action will call it, so that some
  * larger unit of computation, like the enclosing vat, can be shutdown
  * before anything else is corrupted by that corrupted state.
  * See https://github.com/Agoric/agoric-sdk/issues/3434
- * @param {Partial<{elementSchema: Pattern}>} [options]
  * @returns {{
  *  mint: Mint<K>,
  *  issuer: Issuer<K>,
  *  brand: Brand<K>,
  *  displayInfo: DisplayInfo,
  * }}
  */
-const makeIssuerKit = (
-  allegedName,
-  // @ts-expect-error K could be instantiated with a different subtype of AssetKind
-  assetKind = AssetKind.NAT,
-  displayInfo = harden({}),
+export const provideDurableIssuerKit = (
+  issuerBaggage,
   optShutdownWithFailure = undefined,
-  { elementSchema = undefined } = {},
 ) => {
+  const allegedName = issuerBaggage.get('allegedName');
+  const assetKind = issuerBaggage.get('assetKind');
+  const displayInfo = issuerBaggage.get('displayInfo');
+  const elementSchema = issuerBaggage.get('elementSchema');
```

Comment:
If we make this a single call to create an object we can get autocomplete of the names:
```suggestion
  const { allegedName, assetKind, displayInfo, elementSchema }  = issuerBaggage.getAll('allegedName', 'assetKind', 'displayInfo', 'elementSchema');
```

Reply:
This doesn't seem like a big win to me. I typed the names once each and pasted them into the string rather than typing them again. Nothing outside this code should know that these names are being defined or retrieved.